tasks/grunt-postmark-templates.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * grunt-postmark-templates
+ * push templates to a Postmark server for use with SendTemplatedEmail
+ *
+ * https://github.com/wildbit/grunt-postmark.git
+ */
+
+module.exports = function (grunt) {
+  grunt.registerMultiTask('postmark-templates-upload', 'Create or update Postmark templates', function () {
+    var done = this.async();
+    var options = this.options();
+    var template = this.data;
+
+    var serverToken = options.serverToken || grunt.config('secret.postmark.server_token');
+
+    if (!serverToken) {
+      grunt.fail.warn('Missing Postmark server token \n');
+    }
+
+    if (!template.Name) {
```

Comment:
Let's use camel case variables instead of title case. Same goes for `subject`, `htmlBody`, and `textBody` validation. 

I was using the default variables specified from the gruntfile and kept running into validation errors because the variables didn't match.